### PR TITLE
Release notes DietPi 7.2 (initial)

### PR DESCRIPTION
**Pull request by fpetru**
This is the initial PR associated with #488.

### CODE REVIEW

File: docs/releases.md [hunk 1]
```diff
@@ -1,5 +1,70 @@
 # DietPi Releases
 
+## May 2021 (version 7.2 - Beta)
+
+### New optimised software in 7.2
```
Comment (MichaIng) [inline docs/releases.md:5]: Btw, the following is possible when we don't want to have the version string repeated on every header:
```
### New optimised software {: id="new-optimised-software-7.2" }
```
So the visual header and the table of content entry won't have the version string, but the URL when clicking the link or the TOC entry will be `https://dietpi.com/docs/releases/#new-optimised-software-7.2` and hence be unique. That's nice for repeating headers in the software section as well.

Do we want to keep the "optimised" here now that it does not appear in the `dietpi-software` script?

**Comment by MichaIng** [comment]
Nice, I vote for adding and deploying this moreless directly, so I can reference it in a twitter later.

File: docs/releases.md (hunk 1 above)
Comment (fpetru) [reply docs/releases.md:5]: Indeed, I like a lot the way of setting the IDs, and I will apply the changes, both in 7.2 and 7.1. 
The `optimised` keywork is indeed not necessary.

Comment (MichaIng) [reply docs/releases.md:5]: In the headers the in this case (much content) the version string is does not hurt much IMO, but the TOC will look much cleaner without it 😃.

**Comment by MichaIng** [comment]
I removed the dot from the version string in the IDs, as this might cause issues. At least the automatically generated IDs do not contain the dot. Also I changed the main v7.2 header ID so that it won't change after release, i.e. links stay intact.

**Review by MichaIng — APPROVED** [review]

**Pull request merged**